pictor/apache/pictor.conf

```diff
@@ -0,0 +1,13 @@
+  <Directory "/www">
```

Comment:
I think we can get rid of this file, since it is for exposing the deepZoom images for the DPR and/or the Patient WSI.

Reply:
We should probably get rid of it in the other compose configs where we've been porting it in, too.